image_provisioner/playbooks/roles/clone-repos/tasks/main.yaml

```diff
@@ -1,6 +1,7 @@
 ---
 - name: clone svt repo
   git: repo=https://github.com/openshift/svt dest=/root/svt update=yes
+  ignore_errors: yes
```

Comment:
Do a stat before this step and make it conditional if the folder doesn't exist, and get rid of the `ignore_error`